### PR TITLE
[OSDOCS-5337]: Adding metrics sets docs for hosted control planes

### PR DESCRIPTION
**Pull request by lahinson**
Version(s): 4.14

Issue: https://issues.redhat.com/browse/OSDOCS-5337

Link to docs preview: https://63591--docspreview.netlify.app/openshift-enterprise/latest/hosted_control_planes/hcp-managing#hosted-control-planes-metrics-sets_hcp-managing

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by csrwng** [comment]
/lgtm
Thank you!

**Comment by jiezhao16** [comment]
lgtm

File: modules/hosted-control-planes-metrics-sets.adoc [hunk 1]
```diff
@@ -0,0 +1,136 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-managing.adoc
+
+:_content-type: PROCEDURE
+[id="hosted-control-planes-metrics-sets_{context}"]
+= Configuring metrics sets for hosted control planes
+
+Hosted control planes for Red Hat {product-title} creates `ServiceMonitor` resources in each control plane namespace that allow a Prometheus stack to gather metrics from the control planes. The `ServiceMonitor` resources use metrics relabelings to define which metrics are included or excluded from a particular component, such as etcd or the Kubernetes API server. The number of metrics that are produced by control planes directly impacts the resource requirements of the monitoring stack that gathers them.
```
Comment (sabrinajess) [inline modules/hosted-control-planes-metrics-sets.adoc:9]: Is "creates" right here vs. "create"? Wasn't sure if this was meant to be written like a feature or if this was accidental. Ignore if "creates" is accurate.

File: modules/hosted-control-planes-metrics-sets.adoc [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (sabrinajess) [inline modules/hosted-control-planes-metrics-sets.adoc:136]: Are these line breaks intentional?

**Review by sabrinajess — COMMENTED** [review]
Two notes you can ignore if needed. LGTM. Tagging @ousleyp to confirm review.

File: modules/hosted-control-planes-metrics-sets.adoc (hunk 1 above)
Comment (lahinson) [reply modules/hosted-control-planes-metrics-sets.adoc:9]: Yep - "creates" is accurate. The feature name is "hosted control planes for Red Hat OpenShift Container Platform".

File: modules/hosted-control-planes-metrics-sets.adoc (hunk 2 above)
Comment (lahinson) [reply modules/hosted-control-planes-metrics-sets.adoc:136]: Nope! Somehow I didn't notice those. I can remove them. Thanks!

**Comment by ousleyp** [comment]
@sabrinajess Your review LGTM! :)

**Comment by bburt-rh** [comment]
Merged and CP'ed to the 4.14 branch.

**Pull request merged**